tools/dev/goreleaser_and_sign.sh

```diff
@@ -3,18 +3,24 @@ set -eou pipefail
 
 # run from Weaviate Root!
 #
-# This script calls goreleaser and signs + notarize the binaries for macos
+# This script calls goreleaser and signs + notarize the binaries for macos. Thus needed to export few ENV variables
+# to be used in .goreleaser.yaml in the root of the repository.
 
-export GIT_HASH=$(git rev-parse --short HEAD)
+GIT_REVISION=$(git rev-parse --short HEAD)
+GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+BUILD_USER="$(whoami)@$(hostname)"
```

Comment:
let's make it a `ci` here maybe, I'm using this script to build the macOS binaries and attach them to release notes, I think `ci` here would be better (as we have it in `push_docker.sh` script
)

Reply:
Make sense 👍